jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -921,4 +913,97 @@ public static URL toURL(File file) throws MalformedURLException
     {
         return file.toURI().toURL();
     }
+
+    /**
+     * Parse a list of String delimited resources and
+     * return the List of Resources instances it represents.
+     * <p>
+     * Supports glob references that end in {@code /*} or {@code \*}.
+     * Glob references will only iterate through the level specified and will not traverse
+     * found directories within the glob reference.
+     * </p>
+     *
+     * @param delimitedReferences the comma {@code ,} or semicolon {@code ;} delimited
+     * String of resource references.
+     * @param globDirs true if glob references return directories within the glob as well
+     * @return the list of resources parsed from input string.
+     */
+    public static List<Resource> fromList(String delimitedReferences, boolean globDirs) throws IOException
+    {
+        return fromList(delimitedReferences, globDirs, Resource::newResource);
+    }
+
+    /**
+     * Parse a delimited String of resource references and
+     * return the List of Resources instances it represents.
+     * <p>
+     * Supports glob references that end in {@code /*} or {@code \*}.
+     * Glob references will only iterate through the level specified and will not traverse
+     * found directories within the glob reference.
+     * </p>
+     *
+     * @param delimitedReferences the comma {@code ,} or semicolon {@code ;} delimited
+     * String of resource references.
+     * @param globDirs true if glob references return directories within the glob as well
```

Comment:
Isn't this the opposite of the foregoing comment: "Glob references will only iterate through the level specified and will not traverse found directories within the glob reference"?  The intention is that when the wildcard char * is used, deep directory traversal is not supported.

Reply:
No, simple glob is supported, aka only a single asterisk (`*`) which is only the level of the glob itself.

The same as doing a `ls -1 /path/to/look/at/*` (which return all entries at that path location, including the names of the directories)

A full glob support for deep directory searching would be a double asterisk (`**`).
Supporting things like `/opt/**` and returning entries like `/opt/deep/foo.txt`.
Or things like `/code/jetty/**/target/classes` which would return all entries (directories or files) that end in `/target/classes`

Reply:
This behavior does not "traverse found directories" it just returns the directory names found at the glob level.

Reply:
How about "true to return directories in addition to files at the level of the glob"?